docs/usecases/HCP_dataset.rst

```diff
@@ -0,0 +1,580 @@
+
+   But worry not! To get the HCP data, no-one will ever need to compose
+   :command:`clone` commands to RIA stores apart from DataLad itself.
+
+While its layout looks confusing, a RIA store is nothing but a clever storage
+solution, and users never consciously interact with the store to get the HCP
+datasets. On the lowest level, `store.datalad.org <http://store.datalad.org/>`__
+is a directory on a publicly accessible server that holds a great number of such
+bare repositories. The only important aspect of it for this usecase is that
+instead of by their names (e.g., ``100206``), datasets are stored and identified
+via their :term:`dataset ID` (the first two highlighted lines). The hidden section
+above demonstrated how this dataset ID becomes relevant to obtain the dataset.
+Importantly, after cloning a dataset from a RIA store, the dataset clone will
+have its usual file hierarchy and file names.
+
+A RIA store is used, because -- among other advantages -- its layout makes the
+store flexible and scalable. With datasets of sizes like the HCP project,
+especially scalability becomes an important factor. If you are interested in
+finding out why, you can find more technical details on RIA stores, their advantages,
+and even how to create and use one yourself in the use case :ref:`usecase_datastore`.
+
+.. todo::
+
+   Extend the datastore use case
+
+Making the datasets accessible
+""""""""""""""""""""""""""""""
+
+At this point, roughly 1200 nested datasets were created and published to a publicly
+accessible RIA store. This modularized the HCP dataset and prevented performance
+issues that would arise in oversized datasets. In order to make the complete dataset
+available and accessible from one central point, the only thing missing is a
+single superdataset.
+
+For this, a new dataset, ``human-connectome-project-openaccess``, was created.
+It contains a ``README`` file with short instructions how to use it,
+a text-based copy of the HCP projects data usage agreement, -- and each subject
+dataset as a subdataset. The ``.gitmodules`` file [#f1]_ of this superdataset
+thus is impressive. Here is an excerpt::
+
+    [submodule "100206"]
+        path = HCP1200/100206
+        url = ./HCP1200/100206
+        branch = master
+        datalad-id = 346a3ae0-2c2e-11ea-a27d-002590496000
+    [submodule "100307"]
+        path = HCP1200/100307
+        url = ./HCP1200/100307
+        branch = master
+        datalad-id = a51b84fc-2c2d-11ea-9359-0025904abcb0
+    [submodule "100408"]
+        path = HCP1200/100408
+        url = ./HCP1200/100408
+        branch = master
+        datalad-id = d3fa72e4-2c2b-11ea-948f-0025904abcb0
+    [...]
+
+For each subdataset (named after subject IDs), there is one entry. Thus, this
+superdatasets combines all individual datasets to the original HCP dataset
+structure. This (and only this) superdataset is published to a public :term:`GitHub`
+repository that anyone can :command:`datalad clone` [#f4]_.
```

Comment:
Maybe mention that the `url` records are pointless and not needed, because datalad resolves by dataset id from the common store.